FirebaseFunctionsSwift/Tests/Unit/FunctionsTests.swift

```diff
@@ -74,16 +84,52 @@
 //                                                         messaging:nil
 //                                                          appCheck:nil
 //                                                    fetcherService:_fetcherService];
-// }
+  // }
+  var functions: Functions?
+  var functionsCustomDomain: Functions?
+  var fetcherService: GTMSessionFetcherService?
```

Comment:
I think this may have been cleaned up in a future PR but if not, we can assign `GTMSessionFetcherService()` here and remove the force unwrap later.

Reply:
Yep. Done in a future PR.